compiler/src/iree/compiler/InputConversion/Common/QuantizedMatmulToMatmul.cpp

```diff
@@ -88,34 +106,38 @@ struct QuantizedMatmulToMatmul
     int indexOfLhsZpTimesRhsZpTimesKSizeInput = 0;
     Type accElTy = accType.getElementType();
     if (!rhsZpIsConstantZero) {
+      SmallVector<bool> colRedIterator(lhsRank, false);
+      colRedIterator.back() = true;
       Value lhsSums = sumReduceDimensionSubset(builder, lhs, accElTy,
-                                               /*is_reduction=*/{false, true});
+                                               /*is_reduction=*/colRedIterator);
       indexOfLhsSumsInput = addInput(lhsSums, mapToRowDim);
       indexOfRhsZpInput = addInput(rhsZp, mapToNone);
     }
     if (!lhsZpIsConstantZero) {
+      SmallVector<bool> rowRedIterator(lhsRank, false);
+      rowRedIterator[int(batch)] = true;
```

Comment:
style nit: static_cast<int>(batch).

Reply:
thanks @hanhanW for the careful review, I've clearly been sloppy here (C-style or constructor-style casts are quite dangerous indeed, as they are as loose as `reinterpret_cast`, so `static_cast` really matters).